scripts/components/Dialog/InteractionContent.js

```diff
@@ -12,28 +11,6 @@ export default class InteractionContent extends React.Component {
     };
   }
 
-  componentDidUpdate(prevProps) {
```

Comment:
Removing the whole "AudioButton" overhead seems to have worked nicely. Good job!